Build/Scripts/GuardingV2ExtensionsHelper.psm1

```diff
@@ -226,4 +236,30 @@ function Get-BaselineVersion {
     }
 }
 
+<#
+.Synopsis
+    Gets the latest baseline version to use for the breaking change check
+#>
+function Get-LatestBaselineVersionFromArtifacts {
+
+    Import-Module $PSScriptRoot\EnlistmentHelperFunctions.psm1
+
+    [System.Version] $repoVersion = Get-ConfigValueFromKey -Key "RepoVersion"
+
+    if ($repoVersion.Minor -gt 0) {
+        $baselineMajorMinor = "$($repoVersion.Major).$($repoVersion.Minor - 1)"
+    } else {
+        $baselineMajorMinor = "$($repoVersion.Major - 1).5"
```

Comment:
I think if you just do `"$($repoVersion.Major - 1)"` that'll be fine.

`Get-BCArtifactUrl -type Sandbox -country 'W1' -version 19 -select 'Latest'` produces
_https://bcartifacts.azureedge.net/sandbox/19.5.36567.53301/w1_

`Get-BCArtifactUrl -type Sandbox -country 'W1' -version 19.4 -select 'Latest'` produces
_https://bcartifacts.azureedge.net/sandbox/19.4.35398.45489/w1_

Otherwise, what if the .5 version doesn't exist yet.